src/scss/components/_filter-button.scss

```diff
@@ -0,0 +1,21 @@
+@use 'abstracts' as *;
+
+.filter__button {
+  .icon {
+    @include font-size(12px);
+  }
+
+  &:focus:hover {
+    box-shadow: 0 0 0 2px #85b7db;
+  }
+}
+
+.filter__reset {
+  &:focus {
+    box-shadow: none;
+  }
+
+  &:hover {
+    color: $color__white-160 !important;
```

Comment:
I checked this with Tucker when I was making the button components - the reset button should have white text on hover. This override can be removed.